### DISCUSSION TITLE
Application for Global Campus disappers

### DISCUSSION THREAD

**Discussion by Jan-Bober** [community/community#127542] [GitHub Education]
### Select Topic Area

Bug

### Body

I've been trying to get to Global Campus for past half a year. I  have completed all of requirements:
```
To qualify for student benefits, you must:
-Have a GitHub account.
-Be at least 13 years old.
-Be currently enrolled in a degree or diploma granting course of study from a recognized educational institution.
-Be able to provide documentation from your school which demonstrates your current student status.
Before you begin:
-Check that you are using a supported browser, and that location services are not blocked by your browser or platform.
-Complete your GitHub account billing information with your full legal name as it appears on your academic affiliation documentation. (You do not have to add a payment method.)
-Verify  your academic email address on your GitHub account, if your school provides one.
-Secure your GitHub account with two-factor authentication. (We recommend using the GitHub Mobile app.)
-Personalize your public GitHub Profile with your photo, your name, your pronouns, and more.
Generate a README for your public GitHub Profile, customize it with your interests, and publish it for the world to see
```
I have verified school mail and I tried to apply in school, connected to school network and in my home which is close to school.

My application disappers right after sending.

Here a video showing how it looks
https://github.com/community/community/assets/116585135/d2bfff4f-7ae0-42c9-8cf5-f9bd8f02df7d
(poor video quality because max size of video is 10MB)
https://drive.google.com/file/d/1GrGjGfywHcTd34pLA_lhKf6ktTDozf7K/view?usp=sharing
here's video on google drive with better quality
I hope the bug will be fixed or at least my application approved. 
Notice: I tried it month after my friend and his application was approved

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Jan-Bober ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
